File: Python/Day7/Task_seven.py
number = 0
count = 0
multiples_of_4 = 1;
multiples_of_8 = 1;
sum_of_multiples_of_4 = 0;
sum_of_multiples_of_8 = 0;



for _ in range(10):
	number += 1
	if number % 4 == 0:
		for _ in range(5):
			if number / 4 == 1:
				multiples_of_4 *= number
				sum_of_multiples_of_4 += multiples_of_4
			else:
				multiples_of_8 *= number;
				sum_of_multiples_of_8 += multiples_of_8
				
print(sum_of_multiples_of_4,sum_of_multiples_of_8)
					

File: Java/Day7/TaskSeven.java
public class TaskSeven{
public static void main(String[] args){
int numbers = 0;
int multiplesOf4 = 1;
int multiplesOf8 = 1;
int sumOfMultiplesOf4 = 0;
int sumOfMultiplesOf8 = 0;



for(numbers = 1; numbers < 11; numbers ++){
if(numbers % 4 == 0){
for(int count = 1; count < 6; count++){
if(numbers / 4 == 1){
multiplesOf4 *= numbers;
sumOfMultiplesOf4 += multiplesOf4;
}
else{
multiplesOf8 *= numbers;
sumOfMultiplesOf8 += multiplesOf8;
}

}


}

}
System.out.print(sumOfMultiplesOf4 + " " + sumOfMultiplesOf8);




}



}